modules/osdk-building-ansible-operator.adoc

```diff
@@ -67,9 +57,7 @@ shown in the `watches.yaml` file and executes the Ansible role `Memcached`:
 +
 You can optionally customize the following logic in the `watches.yaml` file:
 
-.. Specifying a `role` option configures the Operator to use this specified path
-when launching `ansible-runner` with an Ansible role. By default, the new
-command fills in an absolute path to where your role should go:
+.. Specifying a `role` option configures the Operator to use this specified path when launching `ansible-runner` with an Ansible role. By default, the new command fills in an absolute path to where your role should go:
```

Comment:
I didn't realize _new_ referred to the SDK command referenced earlier, but maybe it's obvious to a reader.